README.md

```diff
@@ -1,3 +1,10 @@
 # Simple irc bot written in golang
 ## This bot is currently going to function on #osdg-iiith on freenode.
 This is a small irc bot written in golang, you're welcome to add functionality to it!
+
+Instructions:
```

Comment:
# Simple irc bot written in golang
## This bot is currently going to function on #osdg-iiith on freenode.
This is a small irc bot written in golang, you're welcome to add functionality to it!

Instructions:
- Install golang: https://golang.org/doc/install
- Setup the GOPATH environment variables: https://golang.org/doc/code.html
- Build and install main and functions packages using the 'go tool': https://golang.org/doc/code.html
- Command Executables are generated in /bin. PATH variable can be reset to execute them.
- If server connection problems arise, check DNS nameserver address in /etc/resolv.conf

//Is this better?